run_script.py

```diff
@@ -0,0 +1,131 @@
+
+import click
+import logging
+import tefla.predict as predict
+import tefla.train_withf as train_withf
+import tefla.predict_withf as predict_withf
+import tefla.metrics as metrices
+import shutil
+import pickle
+import os
+
+# logger = logging.getLogger().addHandler(logging.StreamHandler())
+
+@click.command()
+@click.option('--obj', help='Relative path to model.')
+def main1(obj):
+    return obj+'1'
+
+def copy_and_rename(data_dir,tag):
+    shutil.copy2(data_dir + "predictions/" + tag + "/features.npy", data_dir)
+    os.rename(data_dir + "features.npy", data_dir + tag + "_features.npy")
+
+def analyze_data():
+    lists = []
+    with (open('run_script_logs.pkl', 'rb')) as file:
+        while True:
+         try:
+             lists.append(pickle.load(file))
+         except EOFError:
+            break
+    return lists
+
+def find_best_epoch(list):
+    # filter by best accuracy
+    all_val_accuracy = [obj.val_accuracy for obj in list]
+    max_accuracy=max(all_val_accuracy)
+    all_max_accuracy=[i for i, j in enumerate(all_val_accuracy) if j == max_accuracy]
+    all_max_val_accuracy_list = [list[j] for i, j in enumerate(all_max_accuracy)]
+    # filter by best kappa
+    all_val_kappa = [obj.val_kappa for obj in all_max_val_accuracy_list]
+    max_kappa = max(all_val_kappa)
+    all_max_kappa = [i for i, j in enumerate(all_val_kappa) if j == max_kappa]
+    all_max_val_kappa_list = [all_max_val_accuracy_list[j] for i, j in enumerate(all_max_kappa)]
+    # filter by max training/validation loss ratio
+    all_loss_ratio = [obj.loss_ratio for obj in all_max_val_kappa_list]
+    max_loss_ratio = max(all_loss_ratio)
+    all_loss_ratio_list = [i for i, j in enumerate(all_loss_ratio) if j == max_loss_ratio]
+    all_max_loss_ratio_list=[all_max_val_kappa_list[j] for i, j in enumerate(all_loss_ratio_list)]
+    # filter by min validation loss
+    all_val_loss = [obj.validation_loss for obj in all_max_loss_ratio_list]
+    val_loss=min(all_val_loss)
+    all_val_loss_list = [i for i, j in enumerate(all_val_loss) if j ==val_loss]
+    all_min_val_loss_list = [all_max_loss_ratio_list[j] for i, j in enumerate(all_val_loss_list)]
+    # filter by min training loss
+    all_training_loss= [obj.training_loss for obj in all_min_val_loss_list]
+    train_loss=min(all_training_loss)
+    all_train_loss_list = [i for i, j in enumerate(all_training_loss) if j == train_loss]
+    all_min_train_loss_list = [all_min_val_loss_list[j] for i, j in enumerate(all_train_loss_list)]
+
+    return all_min_train_loss_list
+
+@click.command()
+@click.pass_context
+def call_main(ctx):
+    extract_features=False
+    ask_for_prediction=False
```

Comment:
make this true if you want to be asked for prediction at the end of every training